python-packages/contract_wrappers/stubs/web3/__init__.pyi

```diff
@@ -0,0 +1,55 @@
+from typing import Any, Callable, Dict, List, Optional, Union
+from hexbytes import HexBytes
```

Comment:
Add a blank line here, to separate standard library imports from 3rd party imports.

Reply:
done.